mantidproject/mantid
# Update WAND² IDF and masking for new detector mapping

The order of bank numbers was flipped at the same time the detector mapping was changed so now we mask 2 pixels on all edges.

Also reduce the amount of logging that LoadWAND is doing. `EnableLogging=False`

**To test:**
Look at the diff of old and new WAND IDF and see that bank name have changed.
`diff WAND_Definition_2018_02_20.xml WAND_Definition_2018_04_01.xml `

Try a new file
`silicon_new=LoadWAND('/HFIR/HB2C/IPTS-7776/nexus/HB2C_26632.nxs.h5')`

Check that old files still load correctly with old IDF
`silicon_old=LoadWAND('/HFIR/HB2C/IPTS-7776/nexus/HB2C_26507.nxs.h5')`


*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
